crates/web5/src/apid/dsa/mod.rs

```diff
@@ -1,4 +1,5 @@
 pub mod ed25519;
+pub(crate) mod secp256k1;
```

Comment:
nice!!! I love how we added it, but we aren't committing (yet) to public support (but we will! with more thoughtful consideration), thank you!